### ISSUE TITLE
ahk.image_search(color_variation=n) raises exception

### ISSUE THREAD

**Issue by deshudiosh** [spyoungtech/ahk#199]
It seams that using `color_variation` propery in image search causes error:
`ahk.message.AHKExecutionException: Error occurred in ImageSearch. The error message was: 2
`

This is erroring code:
```
from ahk import AHK

ahk = AHK(executable_path='./AutoHotkey_1.1.36.02/AutoHotkeyU64.exe')

x = ahk.image_search("./icon.jpg", color_variation=50)
print(x)
```

Without `color_variation` it runs smoothly (although icon is not found on the screen)

**Comment by spyoungtech** [comment]
Thanks. This bug appears to be present any time an option of any kind is provided.

I'll have a fix out for this shortly.

**Comment by spyoungtech** [comment]
This is now fixed and released in v1.1.2 🎉

**Comment by deshudiosh** [comment]
This is amazing quality of support. Thank you.
I can confirm that it indeed solved my issue.
